package.json

```diff
@@ -1,15 +1,25 @@
 {
-  "name":       "purescript-echarts",
-  "license":    "MIT",
-  "repository": "git@github.com:slamdata/purescript-echarts.git",
-  "contributors": [
-    "John A. De Goes <john@degoes.net> (http://degoes.net)"
-  ],
-  "devDependencies": {
-    "gulp":             "~3.8.10",
-    "gulp-purescript":  "~0.1.2",
-    "gulp-run":         "~1.6.6",
-    "run-sequence":     "~1.0.2",
-    "gulp-jsvalidate":  "~1.0.1"
+    "name": "purescript-echarts",
+    "license": "MIT",
+    "name":       "purescript-echarts",
+    "license":    "MIT",
+    "repository": "git@github.com:slamdata/purescript-echarts.git",
+    "contributors": [
+        "John A. De Goes <john@degoes.net> (http://degoes.net)"
+    ],
+    "devDependencies": {
+        "ecstatic": "^0.5.8",
+        "express": "^4.11.0",
```

Comment:
express serves examples. I've ran "npm update" to make sure that dependencies up to date.